src/api/student.js

```diff
@@ -20,6 +20,12 @@ app.get('/all', (req, res) => {
     res.successJson(StudentStore.getStudents());
 });
 
+app.patch('/', (req, res) => {
+    addAction('UPDATE_STUDENT', req.body).then(() => {
+        res.successJson(StudentStore.getStudentById(req.body.id));
```

Comment:
Why does that matter? I think it's a waste to have to pull that out of the object when that data is already there, the only reason it's even a patch request it because you already took the post slot for sending the student list.